students/jemielity_kamil/lesson_03_functions/fibonacci_numbers.py

```diff
@@ -0,0 +1,19 @@
+def fib(n):
+    if n == 1:
+        return 1
+    elif n == 0:
+        return 0
```

Comment:
There is no handling for unexpected values like float or negative numbers. Please try to avoid creating endless loop
```
elif n == 0:
    return 0
elif n < 0:
    raise ValueError('Unexpected value: {}'.format(n))
```